content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Bare Metal provisioning on HPE GreenLake using Terraform
+date: 2023-02-26T19:38:34.667Z
+author: Chaita Mylarappachar
+authorimage: /img/chaitra-mylarappachar-192.png
+disable: false
+tags:
+  - hpe-greenlake
+  - terraform
+  - bare-metal-as-a-service
+  - hpegl
+  - BMaaS
+---
+# Introduction
+
+The HPE GreenLake for Private Cloud Enterprise: bare metal (BMaaS) service offering enables you to create dedicated compute instances deployed on a physical IT infrastructure facilitating on-demand scalability, convenience, and agility as a cloud service. 
+Using BMaaS, you can create compute-instances provisioned with specific operating systems, network connections, one or more public SSH keys, and optional network-attached storage volumes.
+The service can be accessed via GUI as well as via public APIs, enabling developers to use an Infrastructure-as-Code tool to build, change, and manage infrastructure in a consistent and repeatable way.
+
+## HPE GreenLake Terraform Provider
```

Comment:
I recommend taking this header out. This way, the next paragraph becomes a part of the overall introduction, which is in-line with the title of your blog post.